Source/Core/Core/IPC_HLE/WII_IPC_HLE.cpp

```diff
@@ -563,6 +563,11 @@ void EnqueueReply_Threadsafe(u32 address, int cycles_in_future)
 	CoreTiming::ScheduleEvent_Threadsafe(cycles_in_future, event_enqueue, address);
 }
 
+void EnqueueReply_Immediate(u32 address)
+{
+	EnqueueEventCallback(address, 0);
+}
```

Comment:
It would be nice if you could refactor this so it calls something with a more relevant name; it isn't really intuitive why you're calling a function named EnqueueEventCallback, and why you're passing it zero.